erpnext_com/www/docs/user/manual/en/stock/quick-stock-balance.md

```diff
@@ -0,0 +1,23 @@
+<!-- add-breadcrumbs -->
+# Quick Stock Balance
+
+Quick Stock Balance is a tool that can be used to quickly retrieve the current stock balance (quantity) and value of an item, in the specified warehouse, on the specified date.
+
+It comes in handy during audits, while verifying stock details before transactions, etc.
+
+To access this tool, go to:
+> Home > Stock > Tools > Quick Stock Balance
+
+## 1. Using the tool
+Item and Warehouse are the basic fields required to fully utilise this tool. Item Barcode can also be used to fetch stock details.
+
+![Quick Stock Balance](/docs/assets/img/stock/quick-stock-balance.png)
+
+You can opt to select a different warehouse  or date and the stock value, along with the quantity, will be retrieved dynamically according to the specified values.
+
+You can also view the Stock Balance report by clicking on 'Stock Balance Report' . It will redirect you to the report with the item and warehouse filters populated.
```

Comment:
```suggestion
You can also view the Stock Balance report by clicking on 'Stock Balance Report'. It will redirect you to the report with the Item and Warehouse filters populated.
```